powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/logging/ObjectStepLogger.java

```diff
@@ -277,12 +291,26 @@ public StepItem getFirstItem(String itemName) {
         return errors;
     }
 
+    /**
+     * Get first {@link StepError} with given name.
+     * @param id Error ID.
+     * @return First {@link StepError} with given ID or {@code null} if logger doesn't contain such error.
+     */
+    public StepError getFirstError(String id) {
+        for (StepError error: errors) {
+            if (error.getId().equals(id)) {
```

Comment:
dtto

Reply:
I inverted the conditions and added null checks.